.github/workflows/deploy.yml

```
@@ -0,0 +1,153 @@
+
+      - name: Build with Gradle
+        run: ./gradlew bootJar
+
+  deploy-on-push:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    name: Build & Deploy (PR merge)
+    steps:
+      # GitHub repository 코드 체크아웃
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)